.github/workflows/link_checker.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Dead Link Checker
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  push:
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.4.1
+        with:
+          args: --exclude-all-private -r 2 './**/*.md' './**/*.html'
+          fail: true
+        env:
```

Comment:
With the schedule set,maybe we don't need a PAT then :)

Reply:
Shall we try the schedule without any others and see if that hits the rate limit? I guess it depends on what has been done on that Github shared runner. If this doesnt work we can try something different